### ISSUE TITLE
New libs due to clipboard stuff

### ISSUE THREAD

**Issue by im-n1** [hrkfdn/ncspot#72]
New libraries are required after sharing functionality has been merged. The readme should be updated. I'm talking about `libxcb-*` packages.

**Comment by phansch** [comment]
I'm not sure if it's related, but after following the README today, the release build still failed with:

```
/usr/bin/ld: cannot find -lxcb-shape                                                                                                     
/usr/bin/ld: cannot find -lxcb-xfixes                                                                                                    
/usr/bin/ld: cannot find -lncursesw
```

I had to install `libncursesw5-dev`, `libxcb-present-dev`, `libxcb-shape0-dev` as well. This is on Ubuntu 18.04.

**Comment by hrkfdn** [comment]
Thanks for the heads up. Okay, so it seems that `xorg-dev` does not include `libxcb` and its development headers. I adapted the wording from https://github.com/aweinstock314/rust-clipboard, but maybe I'll just reword it to indicate the dependency on `libxcb`. The actual names will also vary depending on the distribution, so I'd rather keep it generic.
